README.md

```diff
@@ -22,12 +22,13 @@ You can see the version history [here](RELEASE_NOTES.md).
 - FSharp.Core 4.3.1.0 (F# 3.1)
 - FSharp.Core 4.4.0.0 (F# 4.0)
 - FSharp.Core 4.4.1.0 (F# 4.1)
+- FSharp.Core 4.4.3.0 (F# 4.1 update)
```

Comment:
F# version has got a bit confusing with all the changes, and saying "F# 4.1 update" doesn't help that much. Would you mind changing this to something like this:

- FSharp.Core 4.3.1.0 (F# 3.1 SDK / Visual Studio 2013)
- FSharp.Core 4.4.0.0 (F# 4.0 SDK / Visual Studio 2015)
- FSharp.Core 4.4.1.0 (F# 4.1 SDK / Visual Studio 2017)
- FSharp.Core 4.4.3.0 (F# 10.1 SDK / Visual Studio 2017 15.6)

Reply:
Also, if I understood correctly, you need VS2017 15.6 to be able to build FSharp.Data now, so can you update the line in line 13 to say Visual Studio 2017 15.6+ ?

Reply:
Actually looking at the packagereferences, for .net standard we support FSharp.Core 4.4.1.0 (fsharp.core 4.2.3 nuget package) and 4.4.3.0, maybe join this section with the supported platform one so we have the breakdown of supported fsharp.core versions per platform?

Reply:
Fixed, thanks. Also clarified the design-time environments supported